librz/core/cmd/cmd_regs.c

```diff
@@ -124,6 +124,32 @@ static void print_reg_not_found(const char *arg) {
 	RZ_LOG_ERROR("No such register or register type: \"%s\"\n", rz_str_get(arg));
 }
 
+/**
+ * Register assignment like reg=0x42
+ *
+ * \param core The RzCore instance
+ * \param reg The RzReg instance
+ * \param sync_cb Callback for synchronizing register state in command
+ * \param name Register name
+ * \param val Value
+ * \return success
+ */
+RZ_API bool rz_core_reg_assign_sync(RZ_NONNULL RzCore *core, RZ_NONNULL RzReg *reg, RzCmdRegSync sync_cb, RZ_NONNULL const char *name, ut64 val) {
```

Comment:
is it the callback even used?

Reply:
@imbillow

Reply:
i do not see it called in this function nor passed to any function.

Reply:
@wargio see the definitions of `SYNC_READ` and `SYNC_WRITE`